common/src/main/java/org/wso2/testgrid/common/util/TestGridUtil.java

```diff
@@ -174,8 +174,7 @@ public static Path getTestRunArtifactsDirectory(TestPlan testPlan) throws TestGr
         Product product = deploymentPattern.getProduct();
         int testRunNumber = testPlan.getTestRunNumber();
 
-        String productDir = StringUtil.concatStrings(product.getName(), "_", product.getVersion()
-                , "_", product.getChannel());
+        String productDir = StringUtil.concatStrings(product.getName());
```

Comment:
We do not need the `StringUtil.concatStrings` here

Reply:
:) missed that, Will fix.